build_debian.sh

```diff
@@ -452,6 +452,7 @@ sudo tee $FILESYSTEM_ROOT/etc/sonic/sonic_version.yml > /dev/null <<EOF
 build_version: '${SONIC_IMAGE_VERSION}'
 debian_version: '$(cat $FILESYSTEM_ROOT/etc/debian_version)'
 kernel_version: '$kversion'
+base_os_compatibility_version: ${BASE_OS_COMPATIBILITY_VERSION}
```

Comment:
Is there a plan to display this detail as part of "show version" command output?

Reply:
Hi, yes, I'll share a PR soon

Reply:
https://github.com/Azure/sonic-utilities/pull/1379